src/content/events/2024/bytesize_SIG_meta_omics.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Bytesize: Special Interest Group meta-omics'
+subtitle:  Daniel Lundin (Linnaeus University / Stockholm University), James Fellows Yates (HKI Jena / MPI-EVA Leipzig) and Carson J. Miller (University of Washington)
+type: talk
+startDate: '2024-09-03'
+startTime: '13:00+02:00'
+endDate: '2024-09-03'
+endTime: '13:30+02:00'
+locations:
+  - links: https://kth-se.zoom.us/j/68390542812
+---
+
+In this weeks bytesize Daniel Lundin ([@erikrikarddaniel](https://github.com/erikrikarddaniel)), James Fellows Yates ([@jfy133](https://github.com/jfy133)) and Carson J. Miller ([@carsonjm](https://github.com/carsonjm)) introduce the [Special Interest Group "meta-omics"](https://nf-co.re/special-interest-groups/meta-omics). This group is a place to discuss the subject around _meta_ genomics, _meta_ taxonomics, _meta_ barcoding, _meta_ transcriptomics, _meta_ proteomics and all other _meta_ omics. Some pof the pipelines that would fall into this Special Interest Group are nf-core/ampliseq, nf-core/differentialabundance, nf-core/eager and many more. 
```

Comment:
```suggestion
In this week's bytesize Daniel Lundin ([@erikrikarddaniel](https://github.com/erikrikarddaniel)), James Fellows Yates ([@jfy133](https://github.com/jfy133)) and Carson J. Miller ([@carsonjm](https://github.com/carsonjm)) introduce the [Special Interest Group "meta-omics"](https://nf-co.re/special-interest-groups/meta-omics). This group is a place to discuss the subject around _meta_ genomics, _meta_ taxonomics, _meta_ barcoding, _meta_ transcriptomics, _meta_ proteomics and all other _meta_ omics. Some of the pipelines that would fall into this Special Interest Group are nf-core/ampliseq, nf-core/differentialabundance, nf-core/eager and many more. 
```

Reply:
In my opinion, the core pipelines should include at least Ampliseq, Mag, Metatdenovo and Magmap.

Reply:
```suggestion
In this weeks bytesize Daniel Lundin ([@erikrikarddaniel](https://github.com/erikrikarddaniel)), James A. Fellows Yates ([@jfy133](https://github.com/jfy133)) and Carson J. Miller ([@carsonjm](https://github.com/carsonjm)) introduce the [Special Interest Group "meta-omics"](https://nf-co.re/special-interest-groups/meta-omics).

This group is a place to discuss the subject around _meta_ genomics, _meta_ taxonomics, _meta_ barcoding, _meta_ transcriptomics, _meta_ proteomics and all other _meta_ omics - basically any field that aim to study more than a single organism at the same time. Some of the pipelines that would fall into this Special Interest Group are nf-core/ampliseq, nf-core/taxprofiler, nf-core/mag, nf-core/metatdenovo, and nf-core/magmag, amongst others. 
```

Reply:
I don't know how to suggest a correction on a suggestion but "magmag" should be "magmap".

Reply:
Fixed